README.md

```diff
@@ -77,6 +73,7 @@ for (const i of BigInt.range(0n, 43n)) console.log(i) // 0n to 42n
 
 // With iterator helper proposal
 Number.range(0, Infinity)
+    .values()
```

Comment:
this is an example of poor ergonomics caused by this change. Why is this needed?

Reply:
It is required cause we changed to iterable because iterator helpers doesn't work with iterable classes.

This is the most elegant way in the current limitations and constraints.

Iterator.from(range).take

Or

range[Symbol.iterator]()

is worth than range.values()

Reply:
The most elegant way is to return an iterator directly. This is strictly less elegant/ergonomic, as a tradeoff for whatever benefits are perceived from using an iterable (which i still don’t agree with).

Reply:
There is no change in the main `for ... of ...` case. For the Iterator Helper's case, it is a bit longer but I have tried to make the inconvenient minimal.
A chained call `values()` can be written easily (compare than `Iterator.from` or `[Symbol.iterator]()`). With auto-complete in code editors, it will be even easier to write.

It's acceptable for me to have this extra cost to prevent the #17 problem. And as I said in https://github.com/tc39/proposal-Number.range/issues/41#issuecomment-688386745, the iterator way is preventing to add helper methods in an elegant way.

Reply:
"easily" is subjective, but i think it's pretty objective that "you have to type nothing" is easier than "you have to type something".

I don't see #17 as a problem, and a returned iterator would inherit from some kind of RangeIteratorPrototype, on which helper methods could be added precisely as easily as with a class (which could override/shadow IteratorPrototype methods), so that's an utter non-issue.